### PR TITLE
Fix return type for php 8.2

### PR DESCRIPTION
**Pull request by nicolas2**
fix for php 8.2

### CODE REVIEW

**Comment by papac** [comment]
Hello, @nicolas2 glad to see your PR.
Can you make your commit in a single line with `git rebase`?

TIP:
```
git rebase -i 5.x
```
Fix the last two commits and rework the first commit message in English.

File: src/Auth/Guards/SessionGuard.php [hunk 1]
```diff
@@ -22,6 +22,14 @@ class SessionGuard extends GuardContract
      */
     private array $provider;
 
+    /**
+     * Defines the session_key
+     *
+     * @var array
+     */
+    private string $session_key;
```
Comment (papac) [inline src/Auth/Guards/SessionGuard.php:30]: Where are you going to use the `session_key`?  It's defined but not used

**Review by papac — CHANGES_REQUESTED** [review]

File: src/Auth/Guards/SessionGuard.php (hunk 1 above)
Comment (nicolas2) [reply src/Auth/Guards/SessionGuard.php:30]: Using dynamic properties is now deprecated, retourn error php 8.2

**Review by papac — APPROVED** [review]

**Comment by papac** [comment]
@nicolas2 Thank you for contrib. So I have a current task, we want to add the PostgreSQL Driver [https://snapdev.notion.site/Add-the-PostgreSQL-support-a30cc86e9af24a52b7edadc8252702c8](https://snapdev.notion.site/Add-the-PostgreSQL-support-a30cc86e9af24a52b7edadc8252702c8)

**Pull request closed without merging**